src/main/java/com/neprozorro/service/LotInfoService.java

```diff
@@ -93,18 +93,21 @@ private Predicate inOrEqualLotStatus(Root<LotInfo> root, CriteriaBuilder criteri
         return root.get(column).in(lotStatuses);
     }
 
-    private Predicate betweenOrEqual(Root<LotInfo> root, CriteriaBuilder criteriaBuilder, String column, String value) {
+    private Predicate rangeTotalPrice(Root<LotInfo> root, CriteriaBuilder criteriaBuilder, String column, String value) {
```

Comment:
method that returns Predicate, which has a method test, may be named like greaterThenOrBetween because essentially you are doing this in the method. And check the naming in the class and the project, your naming should be consistent with other namings, so if there are no field names in names of other method you shouldn't add it as well. Or if you have arguments for changing the general naming in the class or the project you should discuss it and make changes everywhere.

Reply:
done